spec/models/category_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe Category, type: :model do
+  let!(:parent) {FactoryBot.create(:category, id: 1)}
```

Comment:
`let` và `let!` khác gì nhau cậu ơi

Reply:
let chỉ chạy khi nào được gọi đến . còn let! được gọi trong một khối before ngầm, do đó nó sẽ chạy trước khi khối it được thực thi